app/javascript/application.js

```diff
@@ -1,6 +1,3 @@
-// Entry point for the build script in your package.json
-import "@hotwired/turbo-rails"
-import "./controllers"
 import React from 'react';
 import ReactDOM from 'react-dom';
 
```

Comment:
- [ ] You really need to double-check and ensure that the React and Redux part of the project is complete. Let's kindly make sure there's a Greeting component. An App component with routes from react-router should lead to the Greeting component. Could you make sure there's a store for state management, an action, and a reducer that uses the URL of the endpoint from the Rails backend? 
- Please at the end double-check if the two apps connect together and random greetings should be displayed when a  request is made on the browser :+1: